README.md

```diff
@@ -249,25 +249,24 @@ described in the following section.
 
 #### The Resource Description Framework (RDF)
 
-[Resource Description Framework](https://www.w3.org/TR/rdf11-concepts/) is a framework for
-describing entities on the web. It uses the Universal Resource Identifier (URI), a generalization
-of the Universal Resource Location (URL), to address resources. This makes RDF exceptionally
-interoperable and extensible.
+The [Resource Description Framework](https://www.w3.org/TR/rdf11-concepts/) is a framework for
+describing ontologies. It uses the [Universal Resource Identifier](https://tools.ietf.org/html/rfc3986)
```

Comment:
`ontologies`.
Can't find the source anymore, but I once read that the semantic web is only defining _one_ ontology, not many.